src/util/classes.ts

```diff
@@ -79,7 +84,7 @@ export const defaultCategories: Category[] = [
   },
   {
     name: ['Comms'],
-    rule: { type: null },
+    rule: { type: 'none' },
```

Comment:
Idk why it thought this should be changed. The `Uncategorized` below has `{ type: null }` too, so I'm pretty sure it was correct. Don't trust the AI.